core/field_dropdown.ts

```diff
@@ -114,30 +113,27 @@ export class FieldDropdown extends Field {
    * @throws {TypeError} If `menuGenerator` options are incorrectly structured.
    */
   constructor(
-      menuGenerator: AnyDuringMigration[][]|Function|Sentinel,
-      opt_validator?: Function, opt_config?: FieldConfig) {
+      menuGenerator: MenuGenerator, opt_validator?: Function,
```

Comment:
nit: can you put each param on a separate line (here and below)? Related to: https://google.github.io/styleguide/jsguide.html#formatting-where-to-break and https://github.com/google/google-java-format/wiki/The-Rectangle-Rule

Reply:
Done!